config/01-ce-auth-defaults.conf

```diff
@@ -68,6 +68,13 @@ AUTH_SSL_SERVER_KEYFILE = /etc/pki/tls/private/localhost.key
 # SciTokens settings
 SCITOKENS_SERVER_AUDIENCE=$(COLLECTOR_HOST)
 
+# Allow READ-level TOKEN authorizations to be generated by the collector.
```

Comment:
- If this comment is true, we should move this configuration to https://github.com/htcondor/htcondor-ce/blob/master/config/04-ce-collector-auth.conf
- If it's needed for both the CE and the central collector, we should move it to https://github.com/htcondor/htcondor-ce/blob/master/config/01-common-auth-defaults.conf
- Otherwise if it's for a site CE's collector, this comment should be clarified

Reply:
The comment's right - maybe the English is just not clear.

That's a CE setting which allows the collector to generate TOKENs.  Will tweak.